### DISCUSSION TITLE
PowerPact Evolution, Air Cooled and output

### DISCUSSION THREAD

**Discussion by M-Struck** [jgyates/genmon#1138] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

V1.19.03

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

Just a question about a PowerPact Evolution, Air Cooled.  I've only had the unit for a couple of months and purchased a kit from PintSize.Me.  We had our first utility failure two nights ago.  During the outage the monitoring page showed RPMs and frequency but anything output related (amps, voltage, etc.) was 0.  The PintSize.Me team wasn't sure why that would be so I am asking here.  The known issues page does talk about it (possibly), just not sure what it means.

**Comment by jgyates** [comment]
@M-Struck 

I would need to see either a screenshot of the web interface while it is running or you would need to submit your logs while it is running to give you some definitive answers. I don't recall the exact differences in the PowerPact vs a normal Evolution, but I do recall that it is a subset of the functionality. The output voltage should not be zero, but the amps could be zero depending on your genmon settings. I will also add the caveat that I have a limited amount of testing with the PowerPact as that is not as popular and I have fewer people who have sent me register dumps for testing for this model. I recall that there are a few things that are not supported on the powerpact, like being able ot read the service due data over modbus. In general if it is not a supported reading then it is disabled rather than reporting zero. 

If you could provide exact info on what you are reading when powering from the generator that would be useful (screenshots and log submissions).

If you want to submit your logs you need to have outbound email setup and working within genmon.

Let me know if you have any questions.

**Comment by M-Struck** [comment]
OK, thanks.  I will try to simulate a utility loss this weekend.

**Reply by jgyates** [reply]
I received some logs from a power pact model. I assume this are from your.

I checked an update that will disable the reading of the L1 and L2 current from modbus as I do not believe it is supported for the PowerPact model. This will also disable the display of the power output as this is derived from the L1 and L2 current. Just update from the About page, the version number has not changed.

I assume the utility voltage being low is you testing.

Let me know if you have any other questions about the power pact operation. Any time you want to submit registers for unique events just send the logs on the about page as I do not have a lot of data on PowerPact and this will help in my testing.

**Reply by M-Struck** [reply, marked as answer]
Yes, the logs were from me testing no utility power.  Just got busy before I could update.  Too bad it’s not supported but I appreciate you checking it out.  Keep up the great work.  Fantastic program with more information than I could get from what Generac provides/sells.

**Reply by M-Struck** [reply]
Just a follow-up on this...should my voltage still be 0?  Had a brief outage last week and during the exercise window over the weekend, voltage remains 0.

**Reply by jgyates** [reply]
The output voltage is not supported either. I just checked in an update that will not show the output voltage gauge or text for Power Pact. The power pact appears to have modbus outputs similar to the Pre Nexus type controllers.
